upgrade/1.2/Stage_0_Prerequisites.md

```diff
@@ -217,6 +218,44 @@ then clone a local working tree and commit appropriate changes to `customization
 
 To prevent any possibility of losing Workload Manager configuration data or files, a back-up is required. Please execute all Backup procedures (for the Workload Manager in use) located in the `Troubleshooting and Administrative Tasks` sub-section of the `Install a Workload Manager` section of the `HPE Cray Programming Environment Installation Guide: CSM on HPE Cray EX`. The resulting backup data should be stored in a safe location off of the system.
 
+<a name="validate_ncn_bios_and_bmc_settings">
+
+### Stage 0.6 - Validate NCN BIOS and BMC Settings
```

Comment:
We are trying to avoid adding more manual steps as much as possible.  Could we have this run in prerequisites.sh?   I'm not sure if we are already having them set the IPMI_PASSWORD in that script, but it could be added.

Reply:
I can move it there, should I also move 0.7 (I just pushed this).

Reply:
That would be great.  0.7 will need to be either before or at the beginning of prerequisites.sh because the test that hit the failure runs in prerequisites.sh.